deploy/openshift4/role.yaml

```diff
@@ -60,3 +60,6 @@ rules:
 - apiGroups: ['']
   resources: [pods/exec]
   verbs: [get, create]
+- apiGroups: ['']
+  resources: [events]
+  verbs: [create, get, list, patch, update, delete]
```

Comment:
This added events resources has some issue for operator patch role, is doing debug.

Reply:
after applying new role.yaml, the ncp-patch-role works well, so there is no issue.